src/serializer/functions.js

```diff
@@ -64,11 +64,54 @@ export class Functions {
   moduleTracer: ModuleTracer;
   writeEffects: Map<FunctionValue, AdditionalFunctionEffects>;
 
-  __generateAdditionalFunctionsMap(globalKey: string) {
-    let recordedAdditionalFunctions: Map<
-      ECMAScriptSourceFunctionValue | AbstractValue,
-      { funcId: string, config?: ReactComponentTreeConfig }
-    > = new Map();
+  __optimizedFunctionEntryOfValue(
+    value: Value
+  ): {
+    value: ECMAScriptSourceFunctionValue | AbstractValue,
+    config?: ReactComponentTreeConfig,
+  } | null {
```

Comment:
I know `null` is creeping in the norm elsewhere, but I'd prefer if we stick to `void` for the core of Prepack.